.github/workflows/release.yml

```diff
@@ -9,32 +9,45 @@ jobs:
   goreleaser:
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      -
-        name: Set up Go
+
+      - name: Set up Go
         uses: actions/setup-go@v2
         with:
           go-version: 1.14
-      -
-        name: Install Protoc
+
+      - name: Install Protoc
         uses: arduino/setup-protoc@v1.1.2
         with:
           version: '3.12.3'
-      -
-        name: Install Protoc-gen-go
+
+      - name: Install Protoc-gen-go
         run: |
           GO111MODULE=off go get github.com/golang/protobuf/protoc-gen-go@v1.4.2
           GO111MODULE=off go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@v0.0.0-20200617041141-9a465503579e
           go mod tidy
-      -
-        name: Run GoReleaser
+
+      - name: Get latest release version number
+        id: get_version
+        uses: battila7/get-version-action@v2
```

Comment:
can you pin this and semver_parser below to specific hash versions? want to make sure we don't have the possibility of a silent update to them that exposes us to code we haven't audited in the release process.

Reply:
Done!